RevenueCatUI/Templates/Components/Packages/Package/PackageComponentView.swift

```diff
@@ -35,6 +44,7 @@ struct PackageComponentView: View {
                     viewModel: self.viewModel.stackViewModel,
                     onDismiss: self.onDismiss
                 )
+                .environment(\.componentViewState, componentViewState)
```

Comment:
This is the important part. This is what tells all children views of the package that they are selected so they can be applied the new styles.